src/common/utils/getWidgetIcon.js

```diff
@@ -6,6 +6,7 @@ import Apps from 'common/widgetIcons/Apps'
 import Calendar from 'common/widgetIcons/Calendar'
 import Camera from 'common/widgetIcons/Camera'
 import Default from 'common/icons/Categories'
+import Image from 'common/icons/Images'
```

Comment:
Should this be the new icon `common/widgetIcons/Image` or are we using the existing Images icon?

Reply:
good catch!